tests/unit/custom_annotations_test.yaml

```
@@ -52,14 +52,26 @@ tests:
 
   # st2auth, st2api, st2stream services do not accept custom annotations
 
-  - it: st2web Service accepts custom annotations
+  - it: st2web, st2auth, st2api ans st2stream Services accepts custom annotations
     template: services.yaml
     documentIndex: 3
     set:
       st2web:
         service:
           hostname: some-host-name
           annotations: *annotations
+      st2api:
+        service:
+          hostname: some-host-name
+          annotations: *annotations
+      st2auth:
+        service:
+          hostname: some-host-name
+          annotations: *annotations
+      st2stream:
+        service:
+          hostname: some-host-name
+          annotations: *annotations
```

removing the `documentIndex` line will make it test all of the services in the file.
The only service we don't want to test here is `st2chatops` since it does not have the same annotations.
Though `st2chatops` is disabled by default, I prefer to make the values explicit that I'm relying on for the test like this:
```suggestion
          annotations: *annotations
      st2chatops:
        enabled: false # exclude st2chatops to only test other services
```